REQUIRE

```diff
@@ -1,4 +1,6 @@
 julia 0.4
+Compat 0.8.5
 Distances 0.3.1
+FixedSizeArrays
 StatsBase 0.9.0
-Compat 0.8.5
+PointClouds
```

Comment:
Yes, PointClouds.jl is actively maintained by FugroRoames. It's a container for large point clouds, and there is no real reason to have it here. My bad.